### PR TITLE
Motion updates and interactive Radio and Intel Reports

### PR DESCRIPTION
**Pull request by calebn**
## Motion Updates
* Fixed page transition issues and no scroll links
* Changed page transitions to be less jarring
* Added common motion type constants
* Added more motion to pages for consistency and emphasis

## Radio
* Added interactive radio with two transmissions with more to come

## Intel Reports
* Created interactive intel report tablet with all intel reports viewable

### CODE REVIEW

File: components/styles/GlobalStyles.styled.js [hunk 1]
```diff
@@ -207,14 +207,12 @@ export const GlobalStyle = createGlobalStyle`
 		grid-template-rows: auto auto 1fr auto;
 		grid-template-areas:
 			'nav'
-			'header'
```
Comment (tr3mulant) [inline components/styles/GlobalStyles.styled.js:210]: I believe removing this grid area has resulted in some of the gutters being shifted, resulting in pages that feel more compressed or elements that feel too close together.

Hopefully these screenshots detail what I'm noticing 

dev_grid_template_3_areas
This is with the header area removed. We're missing the gutter area where the header used to be and we now have a gutter at the bottom of the page after the footer that might be pushing that element up. Makes elements feel too close together.


prod_grid_template_4_areas
This is the current 4 area grid

File: pages/comics.js [hunk 2]
```diff
@@ -1,25 +1,102 @@
-import styled from 'styled-components';
-import MotionFooter from '../components/Footer';
-import MotionHeader from '../components/styles/MotionHeader.styled';
+import styled, { css } from 'styled-components';
+import dynamic from 'next/dynamic';
+const DynamicFooter = dynamic(() => import('../components/Footer'));
 import MotionMain from '../components/styles/MotionMain.styled';
 import { DevRoutes } from '../utils/dev_routes';
```
Comment (tr3mulant) [inline pages/comics.js]: Looks like you removed the DevRoutes functionality (getServerSideProps call) but didn't remove the import. Are we wanting to publish the page at this time?
Things I think we might want to consider are: 
- Adjusting the size of the Radio and the Datapad as they feel kinda small on a desktop screen. Maybe that's just me.
- Publishing with the incomplete Datapad. To me, it looks unfished with the bottom 1/4 just cropped off.

**Comment by tr3mulant** [comment]
Holy cow, big update! You did such killer work with that Radio and the Datapad feature. 
Super stoked to get those out there. 
I really like the sizing for the Cain is Coming banner, it feels perfect.
I pulled the pr to test stuff out and things look good.

File: components/styles/GlobalStyles.styled.js (hunk 1 above)
Comment (calebn) [reply components/styles/GlobalStyles.styled.js:210]: Ah I see what you're saying. I actually kind of hate that gutter (I was already hiding by using a negative margin for the nav so header images wouldn't have awkward spaces above them. I did add on a margin to the default header so it should look a little better space-wise.

File: pages/comics.js (hunk 2 above)
Comment (calebn) [reply pages/comics.js]: I've updated the intel tablet and removed the unnecessary imports.

**Comment by calebn** [comment]
OK @tr3mulant, see what you think now.

**Comment by tr3mulant** [comment]
Man, the IntelReport looks so much better now that it is fully completed. 
Think the sizing is a lot better now as well.
How do we feel about visual indicators regarding the buttons for the IntelReport to reveal their use to end users?
Don't know if a title prop on the button would be good, a tooltip sort of feature, or an on-hover style change that indicates to the user?

**Comment by calebn** [comment]
> Pulled your pr late last night and it was looking pretty good on my laptop other than the buttons on the directional pad being off.

@tr3mulant I've got accessibility fixes for the buttons and nudged them around into a little better position. I've also made some image size tweaks. Apparently using the rem unit doesn't work well for images, switched it to pixels. 

Lastly I made some small updates to the twoUp component to reduce the padding on small screens so it looks a little better.

**Comment by tr3mulant** [comment]
Approved! Thanks for all of your effort.

**Pull request merged**